packages/common/container-definitions/api-report/container-definitions.legacy.alpha.api.md

```diff
@@ -330,8 +330,6 @@ export interface IFluidPackageEnvironment {
     };
 }
 
-export { IGenericError }
-
```

Comment:
> While there are changes to the API reports, there are no actual API changes.

I thought there wouldn't be actual changes - need to follow up on this.

Reply:
You should look to step 4 first - #21451. There are changes because when using our generated rollup (legacy.d.ts) api-extractor see less things than using index.d.ts. It does not seem to handle some things well.
This is one case where it was listing an export where there was not import of definition. In this case the type is `@internal` and a re-export from `core-interfaces`. (I don't know why we have an `@internal` re-exports. Those should all be internally manageable. But that is separate concern.)
I'm back from vacation Saturday morning.

Reply:
> I don't know why we have an @internal re-exports. Those should all be internally manageable. But that is separate concern.
I would bet the re-exports were added for convenience before we did any tagging. We should definitely remove any such cases we find.